rule_packages/c/NoReturn.json

```diff
@@ -0,0 +1,55 @@
+{
+  "MISRA-C-2012": {
+    "RULE-17-10": {
+      "properties": {
+        "obligation": "required"
+      },
+      "queries": [
+        {
+          "description": "Function declared with _noreturn will by definition not return a value, and should be declared to return void.",
+          "kind": "problem",
+          "name": "A function declared with _noreturn shall have a return type of void",
+          "precision": "very-high",
+          "severity": "recommendation",
+          "short_name": "NonVoidReturnTypeOfNoreturnFunction",
+          "tags": []
+        }
+      ],
+      "title": "A function declared with _noreturn shall have a return type of void"
+    },
+    "RULE-17-11": {
+      "properties": {
+        "obligation": "advisory"
+      },
+      "queries": [
+        {
+          "description": "Functions which cannot return should be declared with _Noreturn.",
+          "kind": "problem",
+          "name": "A function without a branch that returns shall be declared with _Noreturn",
+          "precision": "high",
```

Comment:
Could this be `very-high`? When might be miss a live branch?

Reply:
You're right, this should be very high. I was confusing precision with accuracy. It is possible to miss a _dead_ branch -- for instance, a third party library that didn't mark their noreturn functions as such...general halting problem goodness...

Is this worth marking 17-9 as "high" precision, or should 17-9 stay "very high"?

Reply:
I think both as `very-high` is good.